pkg/lib/form-layout.less

```diff
@@ -114,7 +114,10 @@
     grid-column: ~"2 / -1";
     line-height: @widget-height;
     margin: 0;
-    width: auto;
+
+    &:not(.ct-form-layout-relax) {
```

Comment:
The nesting here makes a second huge line of selectors like the above but with the :not() after each one. This is the power and danger of nesting.

In this case, it's needed. But generally, it's good to try to avoid this.

![Spider-Man at work behind a desk, with the famous quote, "With great power comes great responsibility."](https://i.kym-cdn.com/photos/images/original/000/933/851/c8e.jpg)